src/action.rs

```diff
@@ -1,6 +1,31 @@
 //! The different actions that can be done.
 
 use crate::key_code::KeyCode;
+use crate::layout::{Stack, WaitingAction};
+use core::fmt::Debug;
+
+/// A newtype around a custom handler for HoldTap actions.
+#[derive(Copy, Clone)]
+pub struct CustomHandler(pub &'static (dyn Fn(&Stack) -> Option<WaitingAction> + Sync));
```

Comment:
Can be avoided.

Reply:
See the comment above. I needed a newtype to impl Debug, PartialEq and Eq for the new variant.